libfreerdp/core/info.c

```diff
@@ -784,7 +784,7 @@ static void rdp_write_info_packet(rdpRdp* rdp, wStream* s)
 	else
 	{
 		/* user name provided by the expert for connecting to the novice computer */
-		cbUserName = ConvertToUnicode(CP_UTF8, 0, settings->Username, -1, &userNameW, 0) * 2;
```

Comment:
behavioural change, justification?

Reply:
I could be wrong here.. But.
This is original code:
`
	if (!settings->RemoteAssistanceMode)
	{
		cbUserName = ConvertToUnicode(CP_UTF8, 0, settings->Username, -1, &userNameW, 0) * 2;
	}
	else
	{
		/* user name provided by the expert for connecting to the novice computer */
		cbUserName = ConvertToUnicode(CP_UTF8, 0, settings->Username, -1, &userNameW, 0) * 2;
	}
`
The problem is in the same code in both 'if' branches.
I suggested, that in remote assistance mode we should use RedirectionUsername, but i may be wrong, please help me here

Reply:
Not entirely sure, need to test this.

Reply:
Ok, just checked. This change is wrong, the `assistance` file parser does not populate the `RedirectionUsername`

Reply:
I removed the condition, so the original behavior was restored.